server/src/storage/merchants.ts

```diff
@@ -20,11 +20,15 @@
  */
 
 import {MERCHANT_KIND} from '../constants/kinds';
-import {MerchantPayload} from '../interfaces';
-import {add} from './datastore';
+import {Filter, MerchantPayload, MerchantResponse} from '../interfaces';
+import {add, getAll} from './datastore';
 
 const addMerchant = async (merchant: MerchantPayload): Promise<number> =>
   add(MERCHANT_KIND, merchant);
 // TODO(#67): Add checks to prevent adding multiple merchants with the same Firebase UID.
 
-export default {addMerchant};
+const getAllMerchants = async (
+  filters?: Filter[]
```

Comment:
ideally we should also check the filters passed in, e.g. they should have valid property

Reply:
@ifhuang Yep I think we should do field validation in the `services/` layer!

Reply:
Yep! I will add a todo for that but don't think I should add that now based on the current timeline.